src/Mvc/Mvc.Api.Analyzers/src/ActualApiResponseMetadataFactory.cs

```diff
@@ -201,7 +201,7 @@ public static class ActualApiResponseMetadataFactory
         private static ITypeSymbol GetExpressionObjectType(SemanticModel semanticModel, ExpressionSyntax expression, CancellationToken cancellationToken)
         {
             var typeInfo = semanticModel.GetTypeInfo(expression, cancellationToken);
-            return typeInfo.Type;
+            return typeInfo.Type!;
```

Comment:
How are you sure this will not be null? Should the method instead return nullable `ITypeSymbol`?